scripts/batch-compare.sh

```
@@ -61,7 +65,7 @@ run_scenario() {
 
         RESULT=\$(wget -qO- --post-file=/workspace/benchmarks/compatibility/server/behavior/payload-${scenario}.json \\
             --header='Content-Type: application/json' \
-            --header='Authorization: Bearer preloop-system-token' \
+            --header="Authorization: Bearer \$PRELOOP_SYSTEM_TOKEN" \
```

Shellcheck (0.11.0)

[warning] Did you forget to close this double quoted string?

(SC1078)